prov/verbs/src/verbs_init.c

```diff
@@ -493,6 +493,18 @@ int vrb_find_max_inline(struct ibv_pd *pd, struct ibv_context *context,
 	}
 	qp_attr.sq_sig_all = 1;
 
+	if (vrb_gl_data.def_inline_size >= max_inline) {
+		qp_attr.cap.max_inline_data = vrb_gl_data.def_inline_size;
+		qp = ibv_create_qp(pd, &qp_attr);
+		if (qp) {
+			rst = vrb_gl_data.def_inline_size;
```

Comment:
Need to handle the special case of truescale and iwarp here. Both would successfully create qp even if `qp_attr.cap.max_inline_size` is too big. Truescale would return 0 in `qp_attr.cap.max_inline_size`, iwarp would return the actual size there.